tests/accelerators/test_accelerator_connector.py

```diff
@@ -340,7 +336,7 @@ def test_set_devices_if_none_cpu():
     assert trainer.num_devices == 3
 
 
-@pytest.mark.parametrize("strategy", ["ddp2", "dp"])
+@pytest.mark.parametrize("strategy", ["dp"])
```

Comment:
I think parametrization can be removed since there's only one left now.

Reply:
updated